docs/streams/upgrade-guide.html

```diff
@@ -34,71 +34,44 @@ <h1>Upgrade Guide and API Changes</h1>
-        See below a complete list of <a href="#streams_api_changes_200">2.0</a>, <a href="#streams_api_changes_110">1.1</a>,
-        <a href="#streams_api_changes_100">1.0</a>, and <a href="#streams_api_changes_0110">0.11.0</a> API
-        and semantic changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
-        Additionally, Streams API 1.1.x requires broker on-disk message format version 0.10 or higher; thus, you need to make sure that the message
-        format is configured correctly before you upgrade your Kafka Streams application.
-    </p>
-
-    <p>
-        If you want to upgrade from 0.10.1.x to 2.0.x see the Upgrade Sections for <a href="/{{version}}/documentation/#upgrade_1020_streams"><b>0.10.2</b></a>,
-        <a href="/{{version}}/documentation/#upgrade_1100_streams"><b>0.11.0</b></a>,
-        <a href="/{{version}}/documentation/#upgrade_100_streams"><b>1.0</b></a>,
-        <a href="/{{version}}/documentation/#upgrade_110_streams"><b>1.1</b></a>, and
-        <a href="/{{version}}/documentation/#upgrade_200_streams"><b>2.0</b></a>.
-        Note, that a brokers on-disk message format must be on version 0.10 or higher to run a Kafka Streams application version 2.0 or higher.
-        See below a complete list of <a href="#streams_api_changes_0102">0.10.2</a>, <a href="#streams_api_changes_0110">0.11.0</a>,
-        <a href="#streams_api_changes_100">1.0</a>, <a href="#streams_api_changes_110">1.1</a>, and <a href="#streams_api_changes_200">2.0</a>
-        API and semantical changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
-    </p>
-
-    <p>
-        Upgrading from 0.10.0.x to 2.0.0 directly is also possible.
-        Note, that a brokers must be on version 0.10.1 or higher and on-disk message format must be on version 0.10 or higher
-        to run a Kafka Streams application version 2.0 or higher.
-        See <a href="#streams_api_changes_0101">Streams API changes in 0.10.1</a>, <a href="#streams_api_changes_0102">Streams API changes in 0.10.2</a>,
-        <a href="#streams_api_changes_0110">Streams API changes in 0.11.0</a>, <a href="#streams_api_changes_100">Streams API changes in 1.0</a>, and
-        <a href="#streams_api_changes_110">Streams API changes in 1.1</a>, and <a href="#streams_api_changes_200">Streams API changes in 2.0</a>
-        for a complete list of API changes.
-        Upgrading to 2.0.0 requires two rolling bounces with config <code>upgrade.from="0.10.0"</code> set for first upgrade phase
-        (cf. <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-268%3A+Simplify+Kafka+Streams+Rebalance+Metadata+Upgrade">KIP-268</a>).
-        As an alternative, an offline upgrade is also possible.
+        Upgrading from any older verion to 2.0.0 in the online mode requires two rolling bounces.
```

Comment:
I felt that the previous upgrade section of 2.0.0 is not very clear, and felt a bit "scary" (we do not want people to think "oh my so many things to concern, I would not upgrade`). So reword it a bit. But want @mjsax opinion about the changes.

Reply:
Ack, will improve.